src/views/Profile/index.jsx

```
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)